src/frontend/apps/desk/src/features/members/components/ModalAddMembers.tsx

```diff
@@ -48,7 +102,7 @@ export const ModalAddMembers = ({
           color="primary"
           fullWidth
           disabled={!selectedMembers.length}
-          onClick={() => {}}
+          onClick={() => void handleValidate()}
```

Comment:
`onClick={handleValidate}`

Reply:
I need the `void`, eslint is not happy otherwise.